### ISSUE TITLE
URLs should allow @ (e.g. Medium URLs)

### ISSUE THREAD

**Issue by zg** [jcs/lobsters#242]
Markdown URLs are currently broken for Medium links: https://lobste.rs/s/2ksjwu/yahoo_and_freebsd/comments/i5m8nr#c_i5m8nr

Here's what I was trying to write (notice "@zg" in the first URL):

```
I [migrated](https://medium.com/@zg/good-bye-rackspace-hello-digital-ocean-403e7a87228e) from Arch Linux two months ago and have found that it's been the best switch I've made in several years. I'll shamelessly paste some relevant points from a [previous comment](https://lobste.rs/s/cwc1sc/what_are_you_working_on_this_week/comments/5rjeqv#c_5rjeqv) I made:
- Going from rolling release distribution to a more stable environment was a big plus
- FreeBSD has a wealthy amount of [release information](https://www.freebsd.org/releases/) (they’ll tell you what they’re planning and when to expect it)
- The package manager (`pkg`) has been a very simple, yet powerful tool that I’ve gotten used to over the last few weeks
- Packages seem to be easier to setup and are cleaner (I haven’t verified this). I hate that Arch Linux has to have two separate entities, “official packages” and “AUR packages”… This is a frustrating model, in my opinion.
- FreeBSD takes their security very seriously – they have a page dedicated information on security [here](https://www.freebsd.org/security/)
- They have excellent [documentation](https://www.freebsd.org/docs.html)
- They have an excellent [handbook](https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/)
```

**Comment by jcs** [comment]
fixed by 6e6803f205

**Comment by zg** [comment]
:+1:
